Resources/Prototypes/_EE/Entities/Objects/Misc/identification_cards.yml

```
@@ -0,0 +1,19 @@
+# SAN ID
+- type: entity
+  parent: [ IDCardStandard, SolAllianceInfo ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)